numpy/lib/index_tricks.py

```diff
@@ -629,6 +630,15 @@ class ndindex:
     (2, 0, 0)
     (2, 1, 0)
 
+    >>> for index in np.ndindex((3, 2, 1)):
```

Comment:
since this looks very close to the above example on a quick look, you can add a few words above saying you can pass the shape as a tuple.